Dockerfile.spark

```diff
@@ -0,0 +1,29 @@
+FROM gettyimages/spark:2.1.0-hadoop-2.7
+
+RUN apt-get update && apt-get install -yq --no-install-recommends --force-yes \
+    git \
+    openjdk-7-jdk \
+    maven && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV LIVY_BUILD_VERSION livy-server-0.3.0
+ENV LIVY_APP_PATH /apps/$LIVY_BUILD_VERSION
+ENV LIVY_BUILD_PATH /apps/build/livy
```

Comment:
Sorry I wasn't clear. I meant we need to install the following two kernels, besides the kernels already being installed:

* pysparkkernel
* sparkkernel

You can just add `sparkkernel` to the list of kernels to install, and it should just work.

`pysparkkernel` is different in that it should run in a python 2.7 environment whereas `pyspark3kernel` would run in a python 3 environment. Because the container's python installation is python 3, we need to tell livy where to find the python 2 installation in the image (created via virtualenv or anaconda or something else). The way livy finds the two installations is via the env variables i mentioned above.

The point would be to try to show users that one can have two python versions running fine side by side and how to set it up. This is some extra work, and I would be OK with you just adding `sparkkernel` for now and me creating an issue to add `pysparkkernel` to the docker image to tackle later.